backend/app/services/users/utils.py

```diff
@@ -0,0 +1,43 @@
+import json
+
+from app.core.config import settings
+from app.services.users.roles import add_role
+from app.services.users.schema import UserData
+from search.utils import get_json_data
+
+
+def get_user(github_id: int) -> UserData:
+    users: list[dict[str, str]] = get_json_data(settings.USERS_FILE)
+    return UserData(**[user for user in users if user["github_id"] == github_id][0])
+
+
+def is_username_in_muid(username: str, muid: str) -> bool:
+    if not username or not muid:
+        return False
+    return username == muid.split("-", 2)[2]
```

Comment:
`-` can be used in github username, muid.split() can eventually lead to not full username